bench/bench.go

```diff
@@ -254,16 +254,16 @@ func (bm *Benchmark) Report() string {
 	var buffer bytes.Buffer
 
 	indent := ""
+	if !bm.Pubs.HasSamples() && !bm.Subs.HasSamples() {
+		return "No publisher or subscribers. Nothing to report."
```

Comment:
When could this happen?  If it'd be based on a configuration issue, that should be detected up front?  Perhaps "Error:  No publisher or subscriber samples obtained."?